database/004 - add_new_tables_and_columns.sql

```diff
@@ -0,0 +1,84 @@
+CREATE TABLE "AP_Provinces" (
+  "id" BIGSERIAL PRIMARY KEY NOT NULL,
+  "name" character varying(50) NOT NULL,
+  "created_at" timestamp DEFAULT NOW() NOT NULL,
+  "updated_at" timestamp DEFAULT NOW() NOT NULL,
+  "is_deleted" bit DEFAULT '0' NOT NULL
+);
+
+CREATE TABLE "AP_Cantons" (
+  "id" BIGSERIAL PRIMARY KEY NOT NULL,
+  "name" character varying(50) NOT NULL,
+  "province_id" int NOT NULL,
+  "created_at" timestamp DEFAULT NOW() NOT NULL,
+  "updated_at" timestamp DEFAULT NOW() NOT NULL,
+  "is_deleted" bit DEFAULT '0' NOT NULL
+);
+
+CREATE TABLE "AP_Districts" (
+  "id" BIGSERIAL PRIMARY KEY NOT NULL,
+  "name" character varying(50) NOT NULL,
+  "canton_id" int NOT NULL,
+  "created_at" timestamp DEFAULT NOW() NOT NULL,
+  "updated_at" timestamp DEFAULT NOW() NOT NULL,
+  "is_deleted" bit DEFAULT '0' NOT NULL
+);
+
+CREATE TABLE "AP_Directions" (
+  "id" BIGSERIAL PRIMARY KEY NOT NULL,
+  "exact_direction" character varying(50) NOT NULL,
```

Comment:
```suggestion
  "exact_direction" text NOT NULL,
```

Especially with Costa Rican directions 😂